src/v2i-hub/PedestrianPlugin/src/FLIRWebSockAsyncClnSession.cpp

```diff
@@ -6,10 +6,44 @@ using namespace boost::property_tree;
 
 namespace PedestrianPlugin
 {
+    int 
+    FLIRWebSockAsyncClnSession::calculateMinuteOfYear(int year, int month, int day, int hour, int minute, int second) {
+    // Set up current time
+    std::tm currentTime = {};
+    currentTime.tm_year = year - 1900;
+    currentTime.tm_mon = month - 1;
+    currentTime.tm_mday = day;
+    currentTime.tm_hour = hour;
+    currentTime.tm_min = minute;
+    currentTime.tm_sec = second;
+
+    // Set up start of the year
+    std::tm startOfYear = {};
+    startOfYear.tm_year = year - 1900;
+    startOfYear.tm_mon = 0;
+    startOfYear.tm_mday = 1;
+    startOfYear.tm_hour = 0;
+    startOfYear.tm_min = 0;
+    startOfYear.tm_sec = 0;
+
+    // Calculate difference in seconds
+    std::time_t current = std::mktime(&currentTime);
+    std::time_t start = std::mktime(&startOfYear);
+    if (current == -1 || start == -1) {
+        throw std::runtime_error("Failed to compute time difference");
```

Comment:
Can this error be more descriptive. Not sure what this means. What is failing and why?

Reply:
This lets you know that no value was passed to the mktime function or it generated nothing in current and start. Updating the description